src/main/java/org/cyclops/cyclopscore/nbt/path/parse/StringParser.java

```diff
@@ -0,0 +1,71 @@
+package org.cyclops.cyclopscore.nbt.path.parse;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+public class StringParser {
+    @AllArgsConstructor
```

Comment:
I'm trying to migrate away from lombok. Could you remove these annotations and write it as full code?